src/Aspire.Hosting.Kafka/KafkaContainerImageTags.cs

```diff
@@ -12,7 +12,7 @@ internal static class KafkaContainerImageTags
     public const string Image = "confluentinc/confluent-local";
 
     /// <summary>7.7.0</summary>
```

Comment:
```suggestion
    /// <summary>7.7.1</summary>
```

We need to update these summary tags as well because they are linked in our docs.

cc @sebastienros